addons/project/tests/test_project_update.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
+
+from dateutil.relativedelta import relativedelta
+
+from odoo import fields
+from odoo.addons.mail.tests.common import mail_new_test_user
+from odoo.addons.project.tests.test_project_base import TestProjectCommon
+from odoo.exceptions import AccessError
+from odoo.tests import HttpCase, tagged
+from odoo.tests.common import Form, users
+
+@tagged('-at_install', 'post_install')
+class TestProjectUpdateAccessRights(TestProjectCommon):
```

Comment:
Nice ! 🤩

Reply:
Personally I prefer to have one class per file to easily find the tests.
What do you think @IT-Ideas ?